src/containers/portfolio/ProjectSquare.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { NEUTRALS } from '../../core/styles/Colors';
+
+/* styled components */
+const SquareWrapper = styled.div`
+  width: 200px;
+  height: 200px;
+  background-color: ${NEUTRALS.WHITE};
+  margin: 10px;
```

Comment:
Make sure to alphabetize your styles :) highlight the lines and do Shift+Command+P, and then type "sort natural" and press enter. (You can type "sort nat" and that should be enough to bring the package up.)

Reply:
ooh nice one! done